org.eclipse.m2e.maven.runtime/pom.xml

```diff
@@ -20,19 +20,26 @@
 	</parent>
 
 	<artifactId>org.eclipse.m2e.maven.runtime</artifactId>
-	<version>3.9.601-SNAPSHOT</version>
+	<version>3.9.700-SNAPSHOT</version>
 	<packaging>jar</packaging>
 
 	<name>M2E Embedded Maven Runtime (includes Incubating components)</name>
 
+	<repositories>
+		<repository>
+			<id>maven-staging</id>
+			<url>https://repository.apache.org/content/repositories/maven-2125/</url>
+		</repository>
+	</repositories>
+
```

Comment:
This has to be reverted once Maven 3.9.7 is officially released, which should happen maybe today already.

Reply:
Done.